src/contracts/pods/EigenPodManager.sol

```diff
@@ -54,6 +54,12 @@ contract EigenPodManager is Initializable, OwnableUpgradeable, Pausable, IEigenP
     /// @notice Pod owner to deployed EigenPod address
     mapping(address => IEigenPod) public ownerToPod;
 
+    /// @notice The number of pods that have been deployed
+    uint256 public numPods;
+
+    /// @notice The maximum number of pods that can be deployed
+    uint256 public maxPods;
+
```

Comment:
this seems like the least impactful change. we should discuss if existing pausability is sufficient instead of this.

Reply:
I agree with @ChaoticWalrus. We should discuss with @cpdcpd the efficacy of this limit as it has also be pointed out that users could create pods via the site (at a moderate price) and not even use them. Perhaps we should make the UI interaction a bit less available.

Reply:
Agreed offline it's OK to add